### PR TITLE
LOG-2328: DataSink: ensure circleci config mimics that of livemetrics

### PR DESCRIPTION
**Pull request by pedrocunha**
Removing leftovers from old setup

===

Jira story [#LOG-2328](https://deliveroo.atlassian.net/browse/LOG-2328) in project *Logistics*:

### CODE REVIEW

File: config/initializers/statsd.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'datadog/statsd'
 
 STATSD = Datadog::Statsd.new(
-  'localhost',
+  ENV.fetch('STATSD_HOST', 'localhost'),
   8125,
```
Comment (humzashah) [inline config/initializers/statsd.rb]: port as well

File: docker-compose.ci.yml [hunk 2]
```diff
@@ -19,9 +19,6 @@ services:
       - CIRCLE_PROJECT_REPONAME=${CIRCLE_PROJECT_REPONAME}
       - CIRCLE_PROJECT_USERNAME=${CIRCLE_PROJECT_USERNAME}
       - RAILS_ENV=test
-      - REDIS_URL=redis://redis:6379/0
-      - STATSD_HOST=localhost
-      - STATSD_ENV=test
     links:
       - redis
```
Comment (mjkillough) [inline docker-compose.ci.yml]: Do we need this then? Or is it that `REDIS_URL` defaults to the right value already?

File: config/initializers/statsd.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
   tags: [
     "app:#{ENV.fetch('HEROKU_APP_NAME', 'data-sink')}",
```
Comment (humzashah) [inline config/initializers/statsd.rb]: in livemetrics, we have this:

```ruby
"app:#{ENV.fetch('HEROKU_APP_NAME', ENV.fetch('STATSD_APP_NAME', 'live_metrics'))}"
```

so we don't have to set HEROKU_APP_NAME in Hopper; there we just set STATSD_APP_NAME

Comment (pedrocunha) [reply config/initializers/statsd.rb]: For the sake of moving forward and ditching Heroku for good, I will just allow here `STATSD_APP_NAME` and update the environments before deploying

**Pull request merged**